xml/System/SequencePosition.xml

```diff
@@ -126,7 +126,7 @@
       </Parameters>
       <Docs>
         <param name="other">To be added.</param>
-        <summary>To be added.</summary>
+        <summary>Indicates whether the current <see cref="T:System.SequencePosition" /> is equal to another <see cref="T:System.SequencePosition" />. <see cref="T:System.SequencePosition" /> equality does not guarantee that they point to the same location in <see cref="T:System.Buffers.ReadOnlySequence`1" />.</summary>
         <returns>To be added.</returns>
```

Comment:
```suggestion
        <returns><see langword="true" /> if <paramref name="obj" /> has the same value as this instance; otherwise, <see langword="false" />.</returns></returns>
```